iothub/device/src/PnPOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Devices.Client
+{
+    internal class PnPOptions
```

Comment:
Should the client level have a setter for this options object instead of a setter for the model id? That way we can expand it if other pnp options come along

Reply:
We could, but do we expect more pnp options to come along? @prmathur-microsoft

Reply:
I have same question. Isn't that why options exist on the first place?

Reply:
Looks like I wasn't referring updated code, disregard previous comment